src/Symfony/Component/Serializer/Tests/Normalizer/DateTimeNormalizerTest.php

```diff
@@ -208,6 +208,27 @@ public function testDenormalizeUsingTimezonePassedInConstructor()
         $this->doTestDenormalizeUsingTimezonePassedInConstructor();
     }
 
+    /**
+     * testDenormalizeUsingTimezonePassedInDefaultContext.
+     *
+     * @throws \Symfony\Component\Serializer\Exception\ExceptionInterface
+     */
+    public function testDenormalizeUsingTimezonePassedInDefaultContext()
+    {
+        // Test correction of Timezone when timezone information is added in both the date string and the context of the normalizer.
+        $normalizer = new DateTimeNormalizer(
+            [
+                // This is different from Europe times and has NO daylight saving, so tests always pass.
+                DateTimeNormalizer::TIMEZONE_KEY => 'Australia/Brisbane',
+            ]
+        );
+        $this->assertEquals('2016-01-28T01:39:26+10:00', $normalizer->denormalize('2016-01-27T16:39:26+01:00', \DateTimeInterface::class)->format(\DATE_RFC3339)); //Non UTC
```

Comment:
Is it possible to use assertSame() here?

Reply:
Changed tests to check the objects, as that is what the denormalizer returns, so assertSame would not work as expected on objects.